app/controllers/comments_controller.rb

```diff
@@ -19,20 +8,13 @@ def new
   def create
     @user = User.find(params[:user_id])
     @post = Post.find(params[:post_id])
-    @post = Post.find(params[:post_id])
 
     @comment = @post.comments.new(comment_params)
     @comment.author = @user
```

Comment:
- [ ] An attempt to create a comment simply redirects to a page with multiple forms to create comments. This is not the expected behaviour. Please try to fix it.
![image](https://github.com/Zeph180/blogit/assets/46839548/155b40d4-097f-426b-8aa2-441043e6aaf8)